File: GoFizzBuzz/FizzBuzz2.go
package main

import (
  "fmt"
)

func is_divisible(x int, y int) bool {
  return x%y == 0
}

func show_int(x int) string {
  switch {
  case is_divisible(x, 15):
    return "FizzBuzz"
  case is_divisible(x, 3):
    return "Fizz"
  case is_divisible(x, 5):
    return "Buzz"
  default:
    return fmt.Sprintf("%d", x)
  }
}

func main() {
  for i := 0; i <= 100; i++ {
    fmt.Println(show_int(i))
  }
}

File: RustFizzBuzz/Fizz_Buzz/src/FizzBuzz2.rs
fn main() {
  for n in range(1, 101) {
    println(
      if n % 15 == 0 {
        ~"FizzBuzz"
      } else if n % 3 == 0 {
        ~"Fizz"
      } else if n % 5 == 0 {
        ~"Buzz"
      } else {
        n.to_str()
      }
    );
  }
}